src/lua/LuaSpace.cpp

```diff
@@ -150,12 +186,125 @@ static int l_space_spawn_ship(lua_State *l)
 	return 1;
 }
 
+// functions from ShipAiCmd.cpp
+extern int CheckCollision(DynamicBody *dBody, const vector3d &pathdir, double pathdist, const vector3d &tpos, double endvel, double r);
+extern double MaxEffectRad(const Body *body, Propulsion *prop);
+
+/*
+ * Function: PutShipOnTheRoute
```

Comment:
Generally speaking, in English (and our codebase specifically) you drop articles like 'The' and 'An' - this would be `PutShipOnRoute`; if there's a need to clarify which route, you would add a less context-dependent word (e.g. `PutShipOnDirectRoute`). It's convention to strip articles for brevity and style; it's not `SpawnAShip`, it's `SpawnShip`.

Just something that stood out to me that might not be as obvious to a non-native English speaker. :smile:

Reply:
Done!